### PR TITLE
Transferred About us page to new branch

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/images/svg/cyberjaya-pin-location.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="204" height="53" viewBox="0 0 204 53">
```
Comment (oskar-binary) [inline src/images/svg/cyberjaya-pin-location.svg]: aria label and title to all svgs please 🙏

File: src/pages/about/_our-offices.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+                <MapLink
+                    rel="noopener noreferrer"
+                    target="_blank"
+                    href="https://maps.google.com/?q=Lake+Plaza+Dubai&25.079220,55.149569&ll=25.079220,55.149569&z=16"
+                >
+                    <Pinpoint top="30.6%" left="63.7%">
+                        <Oval />
+                        <DubaiWrapper />
+                        {/* <LabelWrapper>{localize('Dubai')}</LabelWrapper> */}
```
Comment (oskar-binary) [inline src/pages/about/_our-offices.js]: can we add a comment as to why the <LabelWrappers /> are commented?

File: src/pages/about/_join-us.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import { localize, LocalizedLink } from 'components/localization'
+
+const JoinUsContainer = styled.section`
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    padding: 4rem 9.7rem;
+    margin: auto;
+    align-items: right;
+    background-image: linear-gradient(to left, #232232, #223122 81%, rgba(0, 0, 0, 0.26) 50%);
```
Comment (mustofa-binary) [inline src/pages/about/_join-us.js]: we can use rgba for this instead of hex

Comment (sara-fs) [reply src/pages/about/_join-us.js]: This section was removed. Probably they will change it. I'll consider it in new implementation. Thanks.

**Pull request merged**